webapi/src/main/twirl/queries/sparql/v1/searchExtendedStandard.scala.txt

```diff
@@ -482,7 +476,7 @@ WHERE {
                 }
 
                 @if(searchCriterion.matchBooleanNegativeTerms.nonEmpty) {
-                    MINUS {
+                    FILTER NOT EXISTS {
```

Comment:
Fuseki has severe performance problems with `MINUS` and often does much better with `FILTER NOT EXISTS`.